### PR TITLE
Use mysql2 ~> 0.3.13

### PR DESCRIPTION
**Pull request by gyozadoc**
Contributed by @sodabrew

### CODE REVIEW

File: installing_thinking_sphinx/ts3.md [hunk 1]
```diff
@@ -13,30 +13,30 @@ title:  Installing Thinking Sphinx
...
+gem install jdbc-mysql -v "~> 5.1.28"
 {% endhighlight %}
 
 Or by adding them to your Gemfile:
 
 {% highlight ruby %}
-gem 'mysql2',          '0.3.12b5', :platform => :ruby
-gem 'jdbc-mysql',      '5.1.28',   :platform => :jruby
+gem 'mysql2',          '~> 0.3.13', :platform => :ruby
+gem 'jdbc-mysql',      '~> 5.1.28',   :platform => :jruby
```
Comment (sodabrew) [inline installing_thinking_sphinx/ts3.md]: Oops, indentation.

File: installing_thinking_sphinx/ts3.md [hunk 2]
```diff
@@ -13,30 +13,30 @@ title:  Installing Thinking Sphinx
...
 gem 'thinking-sphinx', '~> 3.1.0'
 {% endhighlight %}
 
 You can also refer directly to the git repository - but if you're doing this, specifying the version, branch and commit reference is recommended so when you next go to upgrade the gem, it's clear what you were using (and perhaps why).
 
 {% highlight ruby %}
-gem 'mysql2',          '0.3.12b5', :platform => :ruby
-gem 'jdbc-mysql',      '5.1.28',   :platform => :jruby
+gem 'mysql2',          '~> 0.3.13', :platform => :ruby
+gem 'jdbc-mysql',      '~> 5.1.28',   :platform => :jruby
```
Comment (sodabrew) [inline installing_thinking_sphinx/ts3.md]: indentation.

**Comment by sodabrew** [comment]
How do I go about editing this patch via gyozadoc?

**Comment by pat** [comment]
Just pushed the indentation fixes - Gyoza doesn't allow for editing pull requests unfortunately. Thanks for the patch!

**Comment by sodabrew** [comment]
No problem, thank you!

**Pull request merged**